syz-fuzzer/fuzzer.go

```diff
@@ -786,3 +786,20 @@ func kmemleakScan(report bool) {
 		panic(err)
 	}
 }
+
+func hasUserNamespaces() bool {
+	if _, err := os.Stat("/proc/self/ns/user"); err != nil {
+		if os.IsNotExist(err) {
+			Logf(0, "kernel doesn't support user namespaces")
+			return false
+		}
+		if os.IsPermission(err) {
+			Logf(0, "unable to test user namespaces due to permissions")
+			return false
+		}
+		Logf(0, "Failed to stat /proc/self/ns/user: %v", err)
```

Comment:
These errors will be confusing when namespace sandbox is not used (they are visible when syz-manager is started with -debug flag, or when syz-fuzzer is started manually). I think we need to either (1) remove logging or (2) check ipc.DefaultConfig()|FlagSandboxNamespace.